### ISSUE TITLE
ibazel can't locate bazel if not in $PATH

### ISSUE THREAD

**Issue by alexeagle** [bazelbuild/bazel-watcher#252]
a typical frontend developer will do

`npm install --save-dev @bazel/bazel @bazel/ibazel`
without ever doing the Bazel install instructions.

and then
`ibazel run //src:devserver`

but it fails:
```
Error getting Bazel info exec: "bazel": executable file not found in $PATH
Querying for files to watch...
Bazel query failed: exec: "bazel": executable file not found in $PATH
Bazel query failed: exec: "bazel": executable file not found in $PATH
Runing //src:devserver
Error running Bazel exec: "bazel": executable file not found in $PATH
/home/alexeagle/Projects/my-ng-app/node_modules/@bazel/ibazel failed with code 4.
```

**Comment by alexeagle** [comment]
Note, workaround is `npm install -g @bazel/bazel` so that it's on the $PATH

**Comment by alexeagle** [comment]
fixed by https://github.com/bazelbuild/bazel-watcher/pull/275

**Comment by ipula-cstech** [comment]
@alexeagle I had same issue` Error getting Bazel info exec: "bazel": executable file not found in $PATH` but i used `npm install -g @bazel/bazel`..help needed thanks

**Comment by alexeagle** [comment]
do you have latest ibazel with the fix in #275?

**Comment by ipula-cstech** [comment]
> do you have latest ibazel with the fix in #275?

i just use npm command.. it should  install new version.. i will check and let you know

**Comment by ipula-cstech** [comment]
> do you have latest ibazel with the fix in #275?

i ran `ng build` command it gives  `ENOTDIR: not a directory, open 'npm/BUILD.bazel'` error
